ts/store/reducers/wallet/wallets.ts

```diff
@@ -261,7 +262,23 @@ export const cobadgeListVisibleInWalletSelector = createSelector(
   (creditCardListPot): pot.Pot<ReadonlyArray<CreditCardPaymentMethod>, Error> =>
     pot.map(creditCardListPot, creditCardList =>
       creditCardList.filter(
-        cc => cc.pagoPA === false && cc.info.issuerAbiCode !== undefined
+        cc =>
+          cc.pagoPA === false &&
+          cc.info.issuerAbiCode !== undefined &&
+          cc.info.type !== TypeEnum.PRV
+      )
+    )
+);
+
+/**
+ * Return a Privative card list visible in the wallet
+ */
+export const privativeListVisibleInWalletSelector = createSelector(
+  [creditCardListVisibleInWalletSelector],
+  (creditCardListPot): pot.Pot<ReadonlyArray<CreditCardPaymentMethod>, Error> =>
+    pot.map(creditCardListPot, creditCardList =>
+      creditCardList.filter(
+        cc => cc.info.type === TypeEnum.PRV && cc.pagoPA === false
```

Comment:
```suggestion
       isPrivativeCard
```

Reply:
Updated in [441ef7e](https://github.com/pagopa/io-app/pull/2878/commits/441ef7e23a8eed39eb6082c646da4a14633e6423)